client/src/core/TableClass.js

```diff
@@ -68,6 +65,20 @@ function add_child(x) {
   return this;
 }
 
+export function filter_row_by_subj(row, subject) {
+  switch (subject.subject_type) {
```

Comment:
Should have a default case, even if it's to throw an error about an unexpected subject. Will be more useful to future devs than this throwing undefined and that causing a generic error further down the line (or worse, not throwing and becoming a run-time bug :scream:)